tests/e2e/test_api.py

```diff
@@ -237,6 +237,15 @@ def install_ols() -> tuple[str, str, str]:
         ]
     )
 
+    # one of our libs logs a secrets in debug mode which causes the pod
+    # logs beying redacted/removed completely - we need log at info level
+    configmap_yaml = cluster_utils.run_oc(["get", "cm/olsconfig", "-o", "yaml"]).stdout
+    modified_yaml = configmap_yaml.replace(
```

Comment:
can it this be done via YAML manipulation, not on string level (it can fail when whitespaces will be different etc.)